packages/yoastseo/src/languageProcessing/researches/countSentencesFromText.js

```diff
@@ -4,9 +4,10 @@ import sentencesLength from "../helpers/sentence/sentencesLength.js";
 /**
  * Count sentences in the text.
  * @param {Paper} paper The Paper object to get text from.
+ * @param {Researcher} 	researcher 	The researcher to use for analysis.
  * @returns {Array} The sentences from the text.
  */
-export default function( paper ) {
+export default function( paper, researcher ) {
```

Comment:
Since you're adding the `researcher` to the function, you should amend the tests in `packages/yoastseo/spec/languageProcessing/researches/countSentencesFromTextSpec.js` as well.